protoc-gen-bq-json-schema/genjson/run.go

```diff
@@ -0,0 +1,25 @@
+package genjson
+
+import (
+	"google.golang.org/protobuf/compiler/protogen"
+	"google.golang.org/protobuf/reflect/protoregistry"
+)
+
+func Run(gen *protogen.Plugin, config Config) error {
+	var files protoregistry.Files
```

Comment:
why do we need to create a `protoregistry.Files` here? it looks like we don't use it?

Reply:
We don't. Copy-paste leftovers. Removing.